generative_adversarial_networks/restyle-encoder/align_crop.py

```diff
@@ -0,0 +1,145 @@
+"""
+brief: face alignment with FFHQ method (https://github.com/NVlabs/ffhq-dataset)
+author: lzhbrian (https://lzhbrian.me)
+date: 2020.1.5
+note: code is heavily borrowed from 
+    https://github.com/NVlabs/ffhq-dataset
+    http://dlib.net/face_landmark_detection.py.html
+requirements:
+    apt install cmake
+    conda install Pillow numpy scipy
+    pip install dlib
+    # download face landmark model from: 
+    # http://dlib.net/files/shape_predictor_68_face_landmarks.dat.bz2
+"""
+
+import argparse
+import numpy as np
+import os
+import PIL
+import PIL.Image
+import scipy
+import scipy.ndimage
+import dlib
```

Comment:
Is it possible to make it work without dlib? Please refer to the psgan code and get the key points with face_alignment. Make dlib available with the --use_dlib option.
https://github.com/axinc-ai/ailia-models/tree/master/style_transfer/psgan

Reply:
The commit I just pushed should implement the --use_dlib opiton corretly.
I had to modify a bit one file in the psgan project, but it shouldn't have any effect on its original execution.